### PR TITLE
Tweaks .38 Rubber Bullet Damage

### PR DESCRIPTION
**Pull request by DaveTheHeadcrab**
As of right now the Detective's revolver loaded with standard, easy to get .38 rounds will deal 15 brute damage per bullet, plus 50 stamina damage, and a weaken (stun) of length 3 per hit.

For reference:
 - A laser/energy will deal 20 burn damage per shot, is not easily reloadable, and does not incur broken bones, internal organ damage, or bleeding like the detective's revolver does.

 - A disabler beam will deal 36 stamina damage per shot.

 - A stun baton will weaken someone with a length of 7 per whack.

So what does this mean? 6 shots from the revolver will total 90 brute damage in a very short amount of time, plus bleeding, fractures, and internal organ damage. The first shot will down you which means essentially guaranteed follow up shots, and by the time all 6 chambers are empty you'll have plenty of stamina damage racked up to give the detective all the time he'll need to load another set of bullets and put you into hard crit with another 90 brute.

This PR simply adjusts the damage of these bullets from 15 to 5, reducing the chance of fractures and serious long-lasting damage while maintaining the weapon's viability for what it was always meant to do: Aid in self defense. Appropriate changelogging included.

In short:
Revulver op, pls nerf.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
Don't agree with this, but if this is to be done, then whatever damage taken from brute needs to be added to stamina.

**Comment by DaveTheHeadcrab** [comment]
Wholeheartedly disagree. It does more stamina damage than a disabler already on top of an already instant stun.

**Comment by Plotron** [comment]
Well, this will certainly help with trigger happy detectives who tend to give people fractures.
It will make it a rather weak weapon against the most dangerous antags (overfed vampires, changelings, wizards, traitors with holoparasites, nukies).

5 damage sounds good for less-than-lethal rubber ammo, but personally I think it's too much of a drop as it will impact the Blueshield.

The Detective, though? Yeah, he doesn't need that...

**Comment by DaveTheHeadcrab** [comment]
@Plotron 
Well, the Blueshield shouldn't have a revolver period. I'd much rather them have an energy gun, but that's another PR for another day.

**Comment by tigercat2000** [comment]
no fun allowed

**Comment by Spacemanspark** [comment]
Holy shit, 15 brute? I seriously thought it was already 5.
:+1:

**Comment by DaveTheHeadcrab** [comment]
@tigercat2000 
Fun is my mortal enemy.

**Comment by Fox-McCloud** [comment]
@DaveTheHeadcrab 

So do beanbag rounds; the purpose is that two shots keeps someone down for quite a while; this doesn't happen with 50 stamina damage as the life process will tick and reduce it to 48, therefore meaning the second shot will put them at 98 total stamina.

**Comment by DaveTheHeadcrab** [comment]
@Fox-McCloud 

Beanbag rounds have the distinct disadvantage of not having an instant stun associated with them.

**Comment by Fox-McCloud** [comment]
They're also available for use in far more commonly available guns that security, cargo, and science can all access; the revolver, by contrast, is only accessible to two crewmembers.

**Comment by DaveTheHeadcrab** [comment]
@Fox-McCloud 

Conveniently, however, ammo is *just* as available for the revolver as it is for the beanbag shotgun, and the reason beanbag shells take someone down in two hits is because a double-barrel shotgun only holds two shots.

By contrast, the detective's revolver holds six and is capable of reloading much faster than the doube-barrel or any other shotgun.

**Comment by Earthdivine** [comment]
I kinda sorta like this because it means that the new ballistic weapons are more enticing to get rather then just hoping the Detective can put down that Nukie with a Esheild.

**Comment by HereticChurch** [comment]
:+1: from me. I've never really had to rely on raw damage from my revolver as a Blueshield anyway, as by the time I need lethals, I'll have lethals. I would agree with Fox about the annoyance of that third shot for the stamina stun, but the short stun for revolvers is pretty much what I use them for.

I would like to see the Blueshield get something other than a revolver for themself but that's a suggestion for another day.

**Comment by DaveTheHeadcrab** [comment]
@Earthdivine Given that nukie suits have a 50% reduction to ballistic damage received anyhow (And the fact that nukies typically travel in groups) the most powerful aspect of the detective's revolver when facing them is, I argue, the stun and not the damage.

**Comment by Regen1** [comment]
Fuck yes, revolver is so goddamn OP
:+1:

**Comment by tigercat2000** [comment]
by the way to be clear about my opinion
don't really care one way or the other about this PR
(but removing revolver mods totes does scream no fun allowed)

**Comment by DaveTheHeadcrab** [comment]
@tigercat2000 

But that's not what this PR does. (But it totally needs to be removed. Just not within the scope of the PR.)

**Comment by TheDZD** [comment]
@DaveTheHeadcrab 
no fun

**Comment by tigercat2000** [comment]
@DaveTheHeadcrab I know, which is why I wanted to specify that I don't really care either way about this PR, it's kinda meh whatever
